assets/blocks/quiz/answer-blocks/index.js

```diff
@@ -58,8 +66,16 @@ const questionTypes = {
 				'Add a right answer to this question.',
 				'sensei-lms'
 			),
+			noRightAnswerWhitespace: __(
+				'The value of the right answer can not be blank space.',
```

Comment:
I'm not sure we need to show a different message if they've entered whitespace vs. entered nothing. Showing "Add a right answer to this question." seems like it already explains the problem well enough.